### PR TITLE
Add `README.md` to the project

### LINKED ISSUE #59: SwaySwap needs a readme & about section

### PR DESCRIPTION
**Pull request by pixelsbyeryc**
Closes #59.

Adding `README.md` file and `cover.png` to the project root folder.

**Sections:**
- Header
    - Title with ⚡ emoji
    - Short description (is a placeholder for now)
    - Project Badges (has a placeholder badge for now)
    - UI Image (using `cover.png` as the image)
- About the project
- How to install
    - Set up the environment
    - Running the client locally
- Contributing
- License

This is a placeholder to help you guys set up the README in a nice way, feel free to take over this PR.

cc @eureka-cpu

### CODE REVIEW

**Comment by pixelsbyeryc** [comment]
Shoot. I forgot I had more commits. 😦

**Comment by pixelsbyeryc** [comment]
If anyone can guide me through the process of removing the other commits, I'll do so... Sorry about the trouble.

These are commits I submitted to the PR #53. 

cc @eureka-cpu @luizstacio

**Comment by eureka-cpu** [comment]
> If anyone can guide me through the process of removing the other commits, I'll do so... Sorry about the trouble.
> 
> These are commits I submitted to the PR #53.

I'm afraid I don't have a direct answer off hand, but to avoid this in the future you can get a clean branch after making lots of commits for other branches by doing: `git checkout master` then `git reset --hard origin/master` this will put you at the latest commit in master, then you can `git checkout -b <branch name>`. If you have work you did on another branch, just `git stash` and then do the above, then `git stash pop` on your new branch to put your work on that branch instead.

**Comment by pixelsbyeryc** [comment]
Okay, I reverted the commits so only the README.md would go through. At least it's safe to merge or go from there.

I tried [this](https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request) but it didn't work.

**Comment by pixelsbyeryc** [comment]
Here's a preview of what it looks like. I'm leaving placeholders for you guys to populate. ❤️ 

Screen Shot 2022-04-26 at 1 14 29 AM
Screen Shot 2022-04-26 at 1 15 03 AM
Screen Shot 2022-04-26 at 1 15 28 AM

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
-# swayswap-demo
\ No newline at end of file
+<h1 align="center">⚡️ SwaySwap</h1>
+
+<p align="center">Short description</p>
+
+<div align="center">
+    
+[![Open Source](https://badgen.net/badge/icon/Open%20Source%20?icon=github&label&color=black)](https://github.com/FuelLabs/swayswap)
```
Comment (adlerjohn) [inline README.md:7]: For consistency, use shields.io for the badge, and don't center.

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+
+![SwaySwap Interface](cover.png)
+    
+</div>
+
+<br>
```
Comment (adlerjohn) [inline README.md:13]: Don't need `<br>` anywhere, you can just remove them all.

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+### Set up ENV Variables
+
+### Running the Client
+
+#### Go to the `client` folder
+
+```
```
Comment (adlerjohn) [inline README.md:34]: Use `sh` language for these code blocks.
````suggestion
```sh
````

**Comment by pixelsbyeryc** [comment]
Updated according to @adlerjohn's suggestions.

Screen Shot 2022-04-26 at 9 12 29 AM

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,31 @@
...
+[![License](https://img.shields.io/github/license/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+[![Issues Open](https://img.shields.io/github/issues/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+[![Github Forks](https://img.shields.io/github/forks/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+[![Github Stars](https://img.shields.io/github/stars/FuelLabs/swayswap)](https://github.com/FuelLabs/swayswap)
+
+Short description
+    
+![SwaySwap Interface](cover.png)
+
+## About the Project
```
Comment (luizstacio) [inline README.md:11]: Can remove this for now?

File: README.md [hunk 5]
```diff
@@ -1 +1,31 @@
...
+
+About the project
+
+## Getting Started
+
+1. [Set up your environment](#set-up-env-variables)
```
Comment (luizstacio) [inline README.md:17]: Comment this also for now?

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Comment by pixelsbyeryc** [comment]
@luizstacio done! ✌️

**Comment by eureka-cpu** [comment]
lgtm thanks for this @pixelsbyeryc will defer final approval @luizstacio

File: README.md (hunk 4 above)
Comment (pixelsbyeryc) [reply README.md:11]: I commented it out. Do you want me to fully remove it? @luizstacio

File: README.md [hunk 6]
```diff
@@ -1 +1,59 @@
...
+
+## How to Install
+
+1. [Set up your environment](#set-up-env-variables)
+2. [Running the client](#running-the-client)
+
+<br>
+
+### Set up ENV Variables
+
```
[the remark this answers is not in the document]
Comment (pixelsbyeryc) [reply README.md:21]: Added to Issue #62

File: README.md (hunk 5 above)
Comment (pixelsbyeryc) [reply README.md]: Done!

**Review by luizstacio — APPROVED** [review]
LGTM! Thanks 🙏 ⚡️

**Pull request merged**